scripts/setup.py

```
@@ -34,95 +34,78 @@ def init_nx(nx_python_plugin_version: str):
+    else:
+        print(f"Syncing dependencies for profile: {profile}...")
+        run_command(["uv", "sync", "--group", profile])
+
     print("✅ Python environment setup complete.")
 
 def install_custom_py_generator(custom_py_gen_plugin_name: str):
     print("📦 Installing custom Python generator plugin...")
-    
+
     # Create the tools/generators directory structure for the custom generators
     import shutil
```

Ruff (0.12.2)

Redefinition of unused `shutil` from line 4

Remove definition: `shutil`

(F811)